### DISCUSSION TITLE
docs/ninjutsu/low-ranking-ninja/tarot-online

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#19] [General]
# docs/ninjutsu/low-ranking-ninja/tarot-online

好🐟知时节，当春乃发生

https://996.ninja/docs/ninjutsu/low-ranking-ninja/tarot-online

**Comment by nixiliu** [comment]
嘿咋点进去【呈现嵌入代码时出错 PDF 无效】 下载不到

**Reply by MingGH** [reply]
github那个吗，你得下载那个仓库就可以看到了
